src/parser/ParsedQuery.h

```diff
@@ -309,24 +309,23 @@ class ParsedQuery {
 
   typedef std::variant<vector<string>,char> SelectVarsOrAsterisk;
   // Representation of the 'Selector: All (*)' xor 'Selector: (VarName)+'
```

Comment:
// Represents either "all Variables" (Select *) or a list of explitly selected Variables (Select ?a ?b).